sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/SyntheticDataPublisher.java

```diff
@@ -69,26 +80,64 @@
     void setSourceOptions(String sourceOptions);
 
     @Description("PubSub topic to publish to")
-    @Validation.Required
     String getInsertionPipelineTopic();
```

Comment:
Can you change this to `getPubSubTopic` too? `InsertionPipelineTopic` is too generic now since we have the Kafka option too.

Reply:
Ok, done